angular/projects/admin-nrpti/src/app/mines/mines-records-edit/mines-records-edit.component.html

```diff
@@ -68,7 +68,9 @@ <h2 class="border-0 mb-0">Edit record</h2>
           <label for="typeCode">Permit Type</label>
           <select name="typeCode" id="typeCode" formControlName="typeCode" class="form-control">
             <option *ngFor="let typeCode of permitTypes" [ngValue]="typeCode">
-              {{ typeCode === 'OGP' ? 'Original Permit' : 'Permit Amendment' }}
+              {{
```

Comment:
We are repeating here so it could make sense to pull this out into a function referenced above.